src/algos/fl.py

```diff
@@ -41,7 +41,7 @@ def local_train(self, round: int, **kwargs: Any):
         """
         start_time = time.time()
         avg_loss, avg_accuracy = self.model_utils.train(
-            self.model, self.optim, self.dloader, self.loss_fn, self.device
+            self.model, self.optim, self.dloader, self.loss_fn, self.device, malicious_type=self.config.get("malicious_type", "normal"), config=self.config,
```

Comment:
Instead of passing it in the train arguments, we should make it a member of the class by passing it when creating `model_utils`
That way different algos don't have to change their train function explicitly.